plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -256,11 +259,18 @@ export async function createRouter(
     if (scheduler && databaseTaskStore.listStaleTasks) {
       await scheduler.scheduleTask({
         id: 'close_stale_tasks',
-        frequency: { cron: '*/5 * * * *' }, // every 5 minutes, also supports Duration
+        frequency: {
+          cron:
+            options.config.getOptionalString('scaffolder.staleTasks.cron') ??
```

Comment:
Is this really worth having configurable? Checking every five minutes works pretty well for most use cases. I'd be up for making it every minute per default personally, that's still really not a lot of load on the database.

Reply:
Indeed, I thought to make it configurable as checking every 5 minutes i think it's a bit too rare. 
I'd prefer to configure for our case every minute or so.